.circleci/config.yml

```diff
@@ -416,7 +450,28 @@ jobs:
     executor: integration-test-executor
     parallelism: 12
     steps:
-      - run-integration-tests
+      - run-integration-tests:
+          run-code-coverage: true
+          fail-rerun-count: 2
+  code-coverage:
+    executor: build-executor
+    resource_class: medium
+    steps:
+      - attach_workspace:
+          at: ~/
+      - restore-sonar-cache
+      - run:
+          name: Restore Target Directories (Code Coverage)
+          when: always
+          command: |
+            .circleci/scripts/cc-load.sh
```

Comment:
Can we name the file a bit more meaningful, e.g. `codecoverage-load.sh`, or `coverage-load.sh`?

Also maybe, `restore` is a more suitable suffix here.

Reply:
Done